### PR TITLE
Feature-#556_admin_delete_posts

### PR DESCRIPTION
**Pull request by AlbRoehm**
Fix for #556 

my Habitica User-ID: 55fb8bfb-f3ac-4f99-9034-8e062ad6f0e2

Added a check whether the user is a admin and display the delete option if so. Also added a confirmation dialog for message delete. 

The toast is probably unnecessary ...

### CODE REVIEW

File: Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/social/ChatRecyclerViewAdapter.java [hunk 1]
```diff
@@ -355,9 +358,11 @@ public void onClick(View v) {
                     }
 
                     ChatMessage chatMsg = currentMsg;
-                    if (!chatMsg.uuid.equals(uuid)) {
+
+                    if (!chatMsg.uuid.equals(uuid) )
```
Comment (vIiRuS) [inline Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/social/ChatRecyclerViewAdapter.java]: Please leave the curly braces here.

File: Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/social/ChatRecyclerViewAdapter.java [hunk 2]
```diff
@@ -355,9 +358,11 @@ public void onClick(View v) {
...
                         popupMenu.getMenu().findItem(R.id.menu_chat_delete).setVisible(false);
-                    }
+                    if (user.getContributor().getAdmin())
+		                popupMenu.getMenu().findItem(R.id.menu_chat_delete).setVisible(true);
```
Comment (vIiRuS) [inline Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/social/ChatRecyclerViewAdapter.java]: This should also have curly braces.

**Comment by AlbRoehm** [comment]
Changed braces according to request.

**Comment by vIiRuS** [comment]
Thanks!

I noted this as a contribution in your habitica profile.

**Pull request merged**